doc/operation_guides/manual/install-plugins.md

```diff
@@ -15,9 +19,7 @@ If you have modified the `Gemfile.plugins` file, always repeat the following ste
 [openproject@debian]# cd ~/openproject
 [openproject@debian]# bundle install
 [openproject@debian]# bower install
-[openproject@debian]# RAILS_ENV="production" bundle exec rake db:migrate
-[openproject@debian]# RAILS_ENV="production" bundle exec rake db:seed
-[openproject@debian]# RAILS_ENV="production" bundle exec rake assets:precompile
+[openproject@debian]# RAILS_ENV="production" bundle exec rake db:migrate db:seed assets:webpack
```

Comment:
@oliverguenther no asset precompile anymore? Because we advise to let it run in development mode?